src/codegen/cfg.rs

```diff
@@ -337,6 +338,7 @@ pub struct ControlFlowGraph {
     pub ty: pt::FunctionTy,
     pub selector: u32,
     current: usize,
+    pub array_lengths_temps: ArrayLengthVars, //a new member in the cfg that track the temp vars of arrays
```

Comment:
A comment shouldn't say "this is a new member", it's not new once it is merged.

Please put the comment on the line above. The comment should say that these temp var track the length of the arrays